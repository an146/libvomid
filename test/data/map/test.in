123
0 0 10
1 2 123
3 4 123
10 20 234
15 25 345
