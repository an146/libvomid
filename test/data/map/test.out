10 234 15 345 25 123
